5.0/en/0x90-Appendix-A_Glossary.md

```diff
@@ -61,6 +61,7 @@
 * **Time-based OTP** - A method of generating an OTP where the current time acts as part of the algorithm to generate the password.
 * **Time-of-check to time-of-use** (TOCTOU) - A situation where an application checks the state of a resource before using that resource, but the resource's state can be changed between the check and the use. This can invalidate the results of the check and cause a situation where the application performs invalid actions due to this state mismatch.
 * **Threat Modeling** - A technique consisting of developing increasingly refined security architectures to identify threat agents, security zones, security controls, and important technical and business assets.
+* **TLS client authentication**, also called **Mutual TLS** (mTLS) - In a standard TLS connection, a client can use the certificate provided by the server to validate the server's identity. Where LS client authentication is used, the client also uses its own private key and certificate to allow the server to also validate the client's identity.
```

Comment:
Add missing T to "Where LS client"

Reply:
just to clarify :) replace " Where LS client authentication is used" with " Where **T**LS client authentication is used

Reply:
Just for the record I am really excited to see these changes being made. It adds a lot of clarity. Thank you!

Reply:
Fix done. Thank you! :+1: